mkFit/MkStdSeqs.cc

```diff
@@ -502,8 +587,13 @@ void find_and_remove_duplicates(TrackVec &tracks, const IterationConfig &itconf)
   }
   else //regular dupl cleaning
   {
-    find_duplicates(tracks);
-    remove_duplicates(tracks);
+    if(itconf.m_track_algorithm==4)
```

Comment:
This iteration-specific change is quite buried in the code. Could we instead merge the three duplicate removal functions (find_duplicates, find_duplicates_sharedhits, find_duplicates_sharedhits_pixelseed) and use IterationConfig parameters to change whatever is iteration-specific? Merging the functions would also help minimize duplicated code.

Reply:
this was updated to a flag called "require_dupclean_tight". the merging of the functions is left out of this PR, but I agree that this and other similar simplifications are desirable